frontend/src/assets/manual/en-GB/namedentities.md

```diff
@@ -0,0 +1,10 @@
+In some corpora, you have the option to view named entities. If this option is available, a toggle button labeled *Show named entities* is visible in the document overview, above the document content and metadata.
+
+Activating the toggle will show entities of four types:
+- Persons
+- Locations
+- Organizations
+- Miscellaneous
+
+### Disclaimer
+Note that named entities have been extracted through machine learning using [the TextMiNER library](https://github.com/CentreForDigitalHumanities/TextMiNER). As such, the annotated entities should not be taken at face value, and only be used for exploration purposes.
```

Comment:
Good call to have this disclaimer!

Some comments:
- I think it would be good to be a bit more clear about why machine learning means you shouldn't take classifications at face value.
- Technically, nothing in I-analyzer enforces that you use the textminer library, so I wouldn't hard-code that into the manual here. It would make more sense if the library was linked as part of the corpus documentation, like we do for word models.
- Methodologically, I don't agree that the output of machine learning models should only be used for exploration. Automated classifiers are used all the time in research, and it's possible to use their output data responsibly.

```suggestion
### Disclaimer
Named entities are automatically assigned using machine learning. As such, the annotations can include mistakes, and should not be taken at face value.

If a corpus has named entity annotations, the info page for the corpus will provide more information on how the models were trained. We recommend being critical about the accuracy and possible biases of this process if you intend to use named entities in your research.
```